data_structures/hashing/bloom_filter.py

```diff
@@ -0,0 +1,107 @@
+"""
+See https://en.wikipedia.org/wiki/Bloom_filter
+
+The use of this data structure is to test membership in a set.
+Compared to python built-in set() it is more space-efficent.
+In the following example, only 8 bits of memory will be used:
+>>> b = Bloom(size=8)
```

Comment:
Let's avoid single-letter variable names as discussed in CONTRIBUTING.md

Reply:
changed to bloom